nuitka/code_generation/Indentation.py

```diff
@@ -9,9 +9,9 @@
 
 
 def _indentedCode(codes, count):
+    indent = " " * count
```

Comment:
I recall seeing this one. I think that it would be better to pass rather than a count, which is admittedly easy for coding, to pass a string to indent with instead.